### ISSUE TITLE
[Spotify] Audio ads getting through

### ISSUE THREAD

**Issue by interiorchronos** [CHEF-KOCH/CKs-FilterList#40]
### Issue

When applying CHEF-KOCH's Spotify Ad-Filter List to PiHole, audio ads are still getting through, and the client is showing blank banner ads, as the attached screenshot shows. The client is also sometimes getting stuck on trying to play an ad, which is blocked on the list.

![image](https://user-images.githubusercontent.com/40613495/45627914-dbc25780-ba8a-11e8-9b46-15fe515def70.png)

### Reproduction steps

1. Login to the Spotify Windows Store app with PiHole set up and resolving DNS queries
2. Play some songs and wait till an ad comes up. I've noticed they've mainly been Spotify ones.

**Expected Behaviour**
Audio ads do not play

**Comment by CHEF-KOCH** [comment]
Duplicate of https://github.com/CHEF-KOCH/CKs-FilterList/issues/21.
